R/options.R

```diff
@@ -1,21 +1,29 @@
 # Adapted from https://github.com/IRkernel/IRkernel/blob/master/R/options.r
 
 #' pizzarr_option_defaults
```

Comment:
@keller-mark -- see additions here. "parallel_backend" is now the master control on parallelization and there is a separate flag for whether to use it for writing.